parser-library/nt-headers.h

```diff
@@ -38,6 +38,7 @@ const std::uint16_t NUM_DIR_ENTRIES = 16;
 const std::uint16_t NT_OPTIONAL_32_MAGIC = 0x10B;
 const std::uint16_t NT_OPTIONAL_64_MAGIC = 0x20B;
 const std::uint16_t NT_SHORT_NAME_LEN = 8;
+const std::uint16_t SYMTAB_RECORD_LEN = 18;
```

Comment:
D.5 C standard library headers

2 Every C header, each of which has a name of the form name.h, behaves as if each name placed in the standard library namespace by the corresponding cname header is placed within the global namespace scope. It is unspecified whether these names are first declared or defined within namespace scope (3.3.6) of the namespace std and are then injected into the global namespace scope by explicit using-declarations (7.3.3).

3 [ Example: The header <cstdlib> assuredly provides its declarations and definitions within the namespace std. It may also provide these names within the global namespace. The header <stdlib.h> assuredly provides the same declarations and definitions within the global namespace, much as in the C Standard. It may also provide these names within the namespace std. —end example ]